README.md

```diff
@@ -121,9 +136,10 @@ import GeoTIFF from 'geotiff';
 or:
 
 ```html
-<script src="dist-browser/main.js"></script>
+<script async src="https://cdn.jsdelivr.net/npm/geotiff"></script>
 <script>
   console.log(GeoTIFF);
+  // Note: GeoTIFF.Pool will not work
```

Comment:
Why is that exactly? The worker is not available on jsdelivr? Can we upload/cache just one file there?

Reply:
Because the worker has its own bundle, and it is not possible to load a worker from a cross-origin

Reply:
If you wan't to be able to import the bundle from a CDN maybe it is possible with webpack. I can try to do this in another PR (a revert is not enough as we still need a separate build  for NodeJS and browsers)

Reply:
Understood.
And we still need a bundled version for node, right?

Reply:
Yes but it can be achieved with webpack as well.